aeon/io/reader.py

```diff
@@ -212,7 +212,7 @@ def read(self, file):
         specified unique identifier.
         """
         data = super().read(file)
-        data = data[data.event & self.value > 0]
+        data = data[(data.event & self.value) == self.value]
```

Comment:
I don't remember anymore what this change was about, but I feel we should leave any functional changes outside of this PR, which will already be confusing enough with all the major breaking refactoring and reorganisation.